pkg/flowaggregator/flowaggregator.go

```diff
@@ -412,8 +527,9 @@ func (fa *flowAggregator) Run(stopCh <-chan struct{}, wg *sync.WaitGroup) {
 		defer fa.dbExportProcess.Stop()
 	}
 	go fa.flowRecordExpiryCheck(stopCh)
-
+	go fa.watchConfiguration(stopCh)
 	<-stopCh
+	close(fa.updateCh)
```

Comment:
is it useful to close the channel here, and is there any risk of a goroutine trying to write to the channel after it is closed (which would panic)?

Reply:
No, it should be closed under watchConfiguration. I've fixed it.